lib/src/io_streamed_response.dart

```diff
@@ -10,19 +10,19 @@ import 'streamed_response.dart';
 /// An HTTP response where the response body is received asynchronously after
 /// the headers have been received.
 class IOStreamedResponse extends StreamedResponse {
-  final HttpClientResponse _inner;
+  final HttpClientResponse? _inner;
```

Comment:
This is funny. When this class is instantiated in this package, `_inner` is always non-null.

... but it's also exposed as a Type publicly. So switching `inner` to be required in the ctor is a breaking change. 🤷

Reply:
It is not a breaking change to unmigrated code and if it is assumed to be non-null then it is also arguably not breaking (just breaks in a different way).

Reply:
ack

Reply:
@jakemac53 – well. Hrm. Or we just say that `detachSocket` doesn't work if `inner` was `null` when provided.